tests/test_device.py

```diff
@@ -179,22 +179,22 @@ def test_execute(self):
 
             temp = [isinstance(op, qml.operation.CV) for op in queue]
             if all(temp):
-                expval = dev.execute(queue, [qml.expval.X(0, do_queue=False)])
+                expval = dev.execute(queue, [qml.expval(qml.X(0, do_queue=False))])
             else:
-                expval = dev.execute(queue, [qml.expval.PauliX(0, do_queue=False)])
+                expval = dev.execute(queue, [qml.expval(qml.PauliX(0, do_queue=False))])
 
             self.assertTrue(isinstance(expval, np.ndarray))
 
     def test_validity(self):
-        """check that execution throws error on unsupported operations/expectations"""
+        """check that execution throws error on unsupported operations/observables"""
         self.logTestName()
 
         for dev in self.dev.values():
             ops = dev.operations
-            all_ops = {m[0] for m in inspect.getmembers(qml.ops, inspect.isclass)}
+            all_ops = set(qml.ops.__all_ops__)
```

Comment:
this looks pretty simple now. Why was it so complex before?

Reply:
We didn't have any private module attributes before that enabled us to extract available (CV|qubit) (operations|observables). This is the main reason I added the `__all_ops__`, `__all_obs__`, `_qubit_ops_` etc. attributes, which helps simplify these tests (and gets rid of the need to manually inspect the module).